docs/variants/novacustom_ns5x_adl/firmware-update.md

```diff
@@ -0,0 +1,37 @@
+# Firmware update
+
+The following documentation describes the process of Dasharo open-source
+firmware update. The update process may be different, depending on which
+firmware version is currently installed on your device. The currently installed
+firmware version can be checked with the following command in a Linux
+environment:
+
+```shell
+sudo dmidecode -t bios | grep Version
+```
+
+Alternatively, it can be checked in the `BIOS Setup Menu`.
+
+## Version v1.0.0
```

Comment:
From which version I may update to version v1.0.0? This procedure looks like initial deployment.

Either this header is wrong either content of paragraph. Someone coming to this page and thinks:
1. I want to update my NovaCustom NS5x
2. Ok, I have to check my version good, getting some version
3. According this guide one can get results `v1.0.0`
4. One gets information about update to `v1.0.0` this doesn't make sense.

I guess this is WIP.

Reply:
This section has been updated in the unified firmware update doc. I think these points have been adressed